### ISSUE TITLE
TYPO3 v10, Remove deprecated useCacheHash

### ISSUE THREAD

**Issue by cepheiVV** [sabbelasichon/typo3-rector#2734]
# Feature Request

[Deprecation: #88406](https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Deprecation-88406-SetCacheHashnoCacheHashOptionsInViewHelpersAndUriBuilder.html)

I'm using `Typo3LevelSetList::UP_TO_TYPO3_10`.
My Frontend throws an error:

> "Setting typolink.useCacheHash has no effect anymore. Remove the option in all your TypoScript code and Fluid templates"

## Diff

In all TypoScript files:
```diff
-useCacheHash = 1
```

In all Fluid files:
```diff
- <f:link.typolink parameter="{link}" useCacheHash="true" />
+ <f:link.typolink parameter="{link}" />
```

In PHP
```diff
- $lconf['useCacheHash'] = 1;
```

**Comment by sabbelasichon** [comment]
@cepheiVV Would you like to create a PR for that? Would be happy to help you out.

**Comment by cepheiVV** [comment]
Sure, I'd love to do that, if you can guide me thru the process.

**Comment by sabbelasichon** [comment]
Ping me on slack, so we can arrange it.

**Comment by sabbelasichon** [comment]
TypoScript part solved by #3036

**Comment by sabbelasichon** [comment]
No feedback. I am going to close this issue.
